src/engine/scenes/TestScene.cpp

```diff
@@ -0,0 +1,128 @@
+#include "TestScene.hpp"
+
+#include "components/Backpack.hpp"
+#include "components/Model.hpp"
+
+#include "util/GLDebug.hpp"
+
+#include <glm/glm.hpp>
+
+namespace lei3d {
+    TestScene::TestScene() {
+
+    }
+
+    TestScene::~TestScene() {
+
+    }
+
+    void TestScene::LoadObjects() {
+        // load shaders
+        m_MainShader = std::make_unique<Shader>("./data/shaders/transformations.vert", "./data/shaders/transformations.frag");
+
+        // load textures
+        stbi_set_flip_vertically_on_load(true);
+
+        // load mesh from obj file (EVENTUALLY WILL WANT TO USE GTLF FILES INSTEAD)
```

Comment:
So when I uncomment this code, I seem to get a bunch of errors. I'm not quite sure why constructing this model is being a bit funky.